script/run.sh

```diff
@@ -59,25 +59,30 @@ fi
 # CUDA
 if [ "${ALPAKA_CI_INSTALL_CUDA}" == "ON" ]
 then
-    : "${ALPAKA_CUDA_VERSION?'ALPAKA_CUDA_VERSION must be specified'}"
+    : "${ALPAKA_CI_CUDA_VERSION?'ALPAKA_CI_CUDA_VERSION must be specified'}"
 
     if [ "$ALPAKA_CI_OS_NAME" = "Linux" ]
     then
         # CUDA
-        export PATH=/usr/local/cuda-${ALPAKA_CUDA_VERSION}/bin:$PATH
-        export LD_LIBRARY_PATH=/usr/local/cuda-${ALPAKA_CUDA_VERSION}/lib64:${LD_LIBRARY_PATH}
+        export PATH=/usr/local/cuda-${ALPAKA_CI_CUDA_VERSION}/bin:$PATH
+        export LD_LIBRARY_PATH=/usr/local/cuda-${ALPAKA_CI_CUDA_VERSION}/lib64:${LD_LIBRARY_PATH}
         # We have to explicitly add the stub libcuda.so to CUDA_LIB_PATH because the real one would be installed by the driver (which we can not install).
         export CUDA_LIB_PATH=/usr/local/cuda/lib64/stubs/
 
-        if [ "${ALPAKA_CUDA_COMPILER}" == "nvcc" ]
+        if [ "${CMAKE_CUDA_COMPILER}" == "nvcc" ]
         then
             which nvcc
             nvcc -V
         fi
     elif [ "$ALPAKA_CI_OS_NAME" = "Windows" ]
     then
-        export PATH="C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v${ALPAKA_CUDA_VERSION}\bin":$PATH
-        export CUDA_PATH="C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v${ALPAKA_CUDA_VERSION}"
+        export PATH="/c/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v${ALPAKA_CI_CUDA_VERSION}/bin":$PATH
+        export CUDA_PATH="/c/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v${ALPAKA_CI_CUDA_VERSION}"
+
+        which nvcc
+        nvcc -V
+
+        export CUDA_PATH_V${ALPAKA_CI_CUDA_VERSION/./_}="C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v${ALPAKA_CI_CUDA_VERSION}"
```

Comment:
(I'm no Windows expert, so this is pure curiosity:) Is there a reason why we have the `C:/Program Files/` variant here and the `/c/Program Files/` variant above?

Reply:
My guess is that the `run.sh` script is executed in a bash on Windows, which uses `/c/Program Files/` path convensions, whereas every normal Windows tool that will run after bash uses the normal `C:/Program Files` path conventions.

Reply:
`/c/Program Files/` generally doesn't work on Windows. It would sometimes word, like inside certain terminals or some tools may understand it.